Doc/c-api/code.rst

```diff
@@ -30,10 +30,14 @@ bound into a function.
    Return true if *co* is a :ref:`code object <code-objects>`.
    This function always succeeds.
 
-.. c:function:: int PyCode_GetNumFree(PyCodeObject *co)
+.. c:function:: Py_ssize_t PyCode_GetNumFree(PyCodeObject *co)
```

Comment:
It's looks logical if you look at the actual code, but why is it really `Py_ssize_`?
It seems that it was accidentally changed in 02f72b8b938e301bbaaf0142547014e074bd564c. cc @vstinner